docs/configure_service/analise_test.md

```diff
@@ -0,0 +1,91 @@
+
+[`autonomy analyse handlers`](../advanced_reference/commands/autonomy_analyse.md#autonomy-analyse-handlers)
+: This command verifies that all the {{fsm_app}} skills in a local registry (except the explicitly excluded ones) have defined the specified handlers. See the command documentation [here](../advanced_reference/commands/autonomy_analyse.md#autonomy-analyse-handlers).
+
+[`autonomy analyse dialogues`](../advanced_reference/commands/autonomy_analyse.md#autonomy-analyse-dialogues)
+: This command verifies that all the {{fsm_app}} skills in a local registry (except the explicitly excluded ones) have defined the specified dialogues. See the command documentation [here](../advanced_reference/commands/autonomy_analyse.md#autonomy-analyse-dialogues).
+
+[`autonomy analyse service`](../advanced_reference/commands/autonomy_analyse.md#autonomy-analyse-service)
+: This command verifies a service configuration to see if there is any potential issue with it which can cause issues when running the service deployment. See the command documentation [here](../advanced_reference/commands/autonomy_analyse.md#autonomy-analyse-service) and the [on-chain deployment checklist](./on-chain_deployment_checklist.md) for more information.
+
+## Testing
+
+### Unit testing
+
+The `valory/abstract_round_abci` skill packages come with a number of testing tools located in `packages/valory/skills/abstract_round_abci/test_tools` which you can use to write unit tests for your {{fsm_app}}s.
+
+???+ example
+
+    Fetch the `hello_world` agent, which comes with the `hello_world_abci` {{fsm_app}} skill within:
+
+    ```bash
+    autonomy fetch bafybeigspppd7guihobwrhorhht3i47j4qa56tixnicgwtepfeoou54vju
+    mv hello_world hello_world_agent
+    ```
+
+    Look at the unit tests for the `hello_world_abci` skill, located in the folder
+
+    ```
+    ./hello_world_agent/vendor/valory/skills/hello_world_abci/tests/
+    ```
+
+### Integration testing
+
+The `open-aea-test-autonomy` plugin provides several test classes which can be utilized for various Docker integration tests. Here are some examples of the base test classes:
+
+* `aea_test_autonomy.base_test_classes.BaseContractTest`: Comes with utilities required for testing a contract.
+* `aea_test_autonomy.base_test_classes.BaseGanacheContractTest`: Can be used for testing contracts on a local Ganache instance.
+* `aea_test_autonomy.base_test_classes. BaseRegistriesContractsTest`: Launches a local Hardhat node with pre-deployed Autonolas registries contracts image.
+
+### End-to-end testing
+
+The same plugin also provides tools for writing end-to-end tests for agents. The `aea_test_autonomy.base_test_classes.agents.BaseTestEnd2End` class provides an environment for testing agents end to end.
+
+???+ example
+
+    Fetch the `hello_world` agent:
+
+    ```bash
+    autonomy fetch bafybeigspppd7guihobwrhorhht3i47j4qa56tixnicgwtepfeoou54vju
+    mv hello_world hello_world_agent
```

Comment:
Is this really required?

Reply:
The problem is that if the user wants to download the service as well (which unfortunately has the same name) it will raise an exception. Also the flag --alias don't work for services (There is an issue opened).